### PR TITLE
Allow set avatar image on Hugo config.toml

### PR DESCRIPTION
**Pull request by gustavogq**
Hello, I make a small change on baseof.html to allow set avatar image through an avatar attribute on the Hugo config file.

### CODE REVIEW

**Comment by vaga** [comment]
Why you should do that?
You can just add your avatar in `static/avatar.jpg` in your blog directory to replace the default avatar.

**Comment by gustavogq** [comment]
I have done that and for some reason doesn't work for me. Besides that, I think that having the option to easily change the avatar through the config file and doesn't have to rename the file on every time you want to replace de avatar pic (example: if you are trying to find out what pic fits better).

**Comment by TheVetter** [comment]
> I have done that and for some reason doesn't work for me. Besides that, I think that having the option to easily change the avatar through the config file and doesn't have to rename the file on every time you want to replace de avatar pic (example: if you are trying to find out what pic fits better).

Also people like to organize the site for themselves. Options are good. 

I had the same issue and did the [same thing](https://git.thorandrew.com/ScientificPineapple/Scipi-site-theme/commit/f6939c1edfa38648268717d9109b3e523cd4d5cf) with my [copy of the repo](https://git.thorandrew.com/ScientificPineapple/Scipi-site-theme)

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
   </head>
   <body>
     <header class="app-header">
-      <a href="{{ .Site.BaseURL }}"><img class="app-header-avatar" src="{{ "avatar.jpg" | relURL }}" alt="{{ .Site.Params.author | default "John Doe" }}" /></a>
+      <a href="{{ .Site.BaseURL }}"><img class="app-header-avatar" src="{{.Site.Params.avatar | default " avatar.jpg "}}" alt="{{ .Site.Params.author | default "John Doe" }}" /></a>
```
Comment (vaga) [inline layouts/_default/baseof.html]:
```suggestion
      <a href="{{ .Site.BaseURL }}"><img class="app-header-avatar" src="{{ .Site.Params.avatar | default "avatar.jpg" | relURL }}" alt="{{ .Site.Params.author | default "John Doe" }}" /></a>
```

**Comment by gustavogq** [comment]
avatar param definition added to README and relURL wrongly removed reinserted.

**Comment by vaga** [comment]
Thanks! 👍

**Pull request merged**